### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by oaugustus** [appspector/flutter-plugin#26]
AppSpector works fine on flutter debug mode. But when I try generate app build, it break. With error message:

````
FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':generateReleaseRFile'.
> Could not resolve all files for configuration ':releaseCompileClasspath'.
   > Could not find androidx.annotation:annotation:1.1.0.
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':javaPreCompileRelease'.
> Could not resolve all files for configuration ':releaseCompileClasspath'.
   > Could not find androidx.annotation:annotation:1.1.0.
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

BUILD FAILED in 21s
````

My Flutter doctor result is:

`````
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.14.6 18G87, locale pt-BR)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
[✓] Android Studio (version 3.6)
[✓] VS Code (version 1.42.1)



neton-alpha:milks_camp oaugustus$ 
[✓] Connected device (1 available)

• No issues found!
`````

**Comment by Dimdron** [comment]
Hi @oaugustus, thanks for the feedback. I don't know why it happened but to fix this error try to put the next line to `dependencies` block of your `android/app/build.gradle` file
`implementation "androidx.annotation:annotation:1.1.0"`

**Comment by oaugustus** [comment]
I had previously placed this line, but the error is the same:

`````
dependencies {
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
    implementation "androidx.annotation:annotation:1.1.0"
}

**Comment by oaugustus** [comment]
When I open the project on Android Studio, Flutter > Open Android Module in Android Studio, and generate build relase from Build > Generate Signed Bundle / APK the build works without error.

**Comment by Dimdron** [comment]
Got it. Did you experience the error in VS Code? I just want to know how to reproduce it.
Or it was Android Studio > Build > Flutter > Build APK?

**Comment by oaugustus** [comment]
The error occurs on terminal flutter build apk --release

**Comment by Dimdron** [comment]
I found the missed declaration of google repository with the androidx packages. It has been fixed in plugin v0.3.0
